assets/css/pre_fare_v2.scss

```diff
@@ -1,6 +1,7 @@
 @import url("https://rsms.me/inter/inter.css");
 
 @import "colors";
+@import "fonts_inter";
```

Comment:
This shouldn't be necessary. We import it via the URL on line 1.

Reply:
Unfortunately, the `solari/departure_time` file has special mixins that are in the fonts file 🙄